com.reprezen.swagedit.core/src/com/reprezen/swagedit/core/hyperlinks/JsonReferenceHyperlinkDetector.java

```diff
@@ -37,7 +37,7 @@
 
     @Override
     protected boolean canDetect(JsonPointer pointer) {
-        return pointer != null && pointer.toString().endsWith("$ref");
+        return pointer != null && (pointer.toString().endsWith("$ref") || pointer.toString().endsWith("operationRef"));
```

Comment:
`(pointer.toString().endsWith("$ref")` is a standard reference, while the one which `.endsWith("operationRef")` is not. How about extracting the check for the "operationRef" in a hook method to make it clear that it's a customization?